### PR TITLE
fusion_info: update return block in documentation fragment

### PR DESCRIPTION
**Pull request by andreepyro**
##### SUMMARY
Fixing recent ansible-lint issues by updating return block in documentation fragment.

##### ISSUE TYPE
- Bugfix Pull Request

##### COMPONENT NAME
- fusion_info

### CODE REVIEW

**Comment by andreepyro** [comment]
@sdodsley please approve to unblock other PRs

**Review by sdodsley — DISMISSED** [review]

**Comment by andreepyro** [comment]
@sdodsley Why not to fix the syntax itself as proposed in this PR?

**Comment by sdodsley** [comment]
@andreepyro I'd like to stick with `main` and if that breaks something we check with the core team and then, if necessary, make code changes.

**Pull request closed without merging**
